<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PY ACESSORIA - CADASTRO DE IMÓVEIS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PY ACESSORIA - CADASTRO DE IMÓVEL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>591</width>
      <height>591</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="title">
     <string>Cadastro/Consulta</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>541</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Preco">
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Preço:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Observações:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_Obs"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Condições:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Características:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_Cond"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QTextEdit" name="textEdit_Caract"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Descrição:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QTextEdit" name="textEdit_Desc"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>539</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEdit_Bairro"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Cidade:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Estado:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cb_TipoNeg"/>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_Comp"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Endereço:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="cb_TipoImovel"/>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="cb_Status"/>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="lineEdit_End"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_Num">
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Bairro:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_Cidade"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tipo de Negociação</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_Cep">
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Complemento:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEdit_Est"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Tipo de Imóvel:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Número:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CEP:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>238</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Identificação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ID"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A identificação é cadastrada automáticamente!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_Consultar">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>560</y>
       <width>168</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Cadastrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ID</tabstop>
  <tabstop>bt_Consultar</tabstop>
  <tabstop>cb_TipoNeg</tabstop>
  <tabstop>cb_Status</tabstop>
  <tabstop>lineEdit_End</tabstop>
  <tabstop>lineEdit_Num</tabstop>
  <tabstop>lineEdit_Comp</tabstop>
  <tabstop>lineEdit_Cep</tabstop>
  <tabstop>lineEdit_Est</tabstop>
  <tabstop>lineEdit_Cidade</tabstop>
  <tabstop>lineEdit_Bairro</tabstop>
  <tabstop>cb_TipoImovel</tabstop>
  <tabstop>textEdit_Desc</tabstop>
  <tabstop>textEdit_Caract</tabstop>
  <tabstop>lineEdit_Preco</tabstop>
  <tabstop>lineEdit_Obs</tabstop>
  <tabstop>lineEdit_Cond</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
